plump-sexp.lisp

```diff
@@ -73,6 +77,13 @@ Alternatively a pathname, stream or string may be passed as well, which will be
 
 (defgeneric serialize (node)
   (:documentation "Serialize the given node into a SEXP form.")
+  (:method ((node xml-header))
+    (list
+     (cons :!HEADER
```

Comment:
```suggestion
     (list* :!HEADER
```
When prepending an element to a list it is better to use `list*` rather than `cons`.